.circleci/config.yml

```diff
@@ -66,6 +66,37 @@ jobs:
           path: packages/ml-sdk/test-results
       - store_test_results:
           path: packages/platform-sdk/test-results
+  integration_test:
+    parameters:
+      os:
+        type: executor
+      node-version:
+        type: string
+    executor: << parameters.os >>
+    working_directory: *working_directory
+    steps:
+      - checkout
+      - restore_cache: *restore_yarn_cache
+      - run:
+          name: Installing dependencies
+          command: yarn install --frozen-lockfile
+      - save_cache: *save_yarn_cache
+      - run:
+          name: Use environment variables
+          command: |
+            echo $IT_CTP_PROJECT_KEY
+            echo $IT_CTP_CLIENT_SECRET
+            echo $IT_CTP_CLIENT_ID
+      - run:
+          name: Running integration tests
+          command: yarn test --coverage --testPathPattern='packages/platform-sdk/test/integration-tests'
```

Comment:
Test coverages shouldn't be test specific but as a collection of the whole tests suites, even though the tests runs one at a time, the coverage should be an aggregate of all the tests.

Reply:
it's still WIP
I'm trying to find a way to combine the coverage for both kind of tests

Reply:
I'll let you know when you can review again the PR